administration/package.json

```diff
@@ -3,56 +3,56 @@
-    "protobufjs": "^6.11.2",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
+    "protobufjs": "^7.1.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "react-flip-move": "^3.0.4",
-    "react-router": "^6.2.1",
-    "react-router-dom": "^6.2.1",
-    "react-scripts": "^4.0.0",
-    "styled-components": "^5.3.3",
-    "web-vitals": "^2.1.4",
+    "react-router-dom": "6.2.1",
+    "react-scripts": "^5.0.1",
+    "styled-components": "^5.3.5",
+    "web-vitals": "^3.0.2",
     "whatwg-fetch": "^3.6.2"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/react": "^12.1.3",
-    "@testing-library/user-event": "^13.5.0",
+    "@graphql-codegen/cli": "2.12.0",
+    "@graphql-codegen/typescript": "2.7.3",
+    "@graphql-codegen/typescript-react-apollo": "3.3.3",
+    "@graphql-codegen/typescript-operations": "2.5.3",
     "@types/blob-stream": "^0.1.30",
-    "@types/jest": "^27.4.0",
-    "@types/node": "^17.0.18",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "@types/react-flip-move": "^2.9.12",
-    "@types/react-router-dom": "^5.3.3",
-    "@types/styled-components": "^5.1.23",
-    "apollo": "^2.33.9",
+    "@types/jest": "^29.0.3",
+    "@types/node": "^18.7.18",
+    "@types/react": "^18.0.20",
+    "@types/react-dom": "^18.0.6",
+    "@types/react-router-dom": "5.3.3",
+    "@types/styled-components": "^5.1.26",
+    "apollo": "^2.34.0",
     "run-script-os": "^1.1.6",
-    "typescript": "^4.5.5"
+    "protobufjs-cli": "^1.0.2",
+    "typescript": "^4.8.3"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "lint": "eslint .",
-    "generate-graphql": "apollo client:codegen --target=typescript --passthroughCustomScalars --customScalarsPrefix=GQL_",
+    "generate-graphql": "graphql-codegen --config graphql-codegen.yml",
     "generate-protobuf": "run-script-os",
-    "generate-protobuf:linux": "mkdir -p src/generated && pbjs -t static-module -w es6 -o src/generated/compiled.js ../specs/card_activate_model.proto --force-long && pbts -o src/generated/compiled.d.ts src/generated/compiled.js",
-    "generate-protobuf:win32": "mkdir src\\generated | pbjs -t static-module -w es6 -o src\\generated\\compiled.js ..\\specs\\card_activate_model.proto --force-long | pbts -o src\\generated\\compiled.d.ts src\\generated\\compiled.js",
-    "postinstall": "npm run generate-protobuf"
+    "generate-protobuf:linux": "mkdir -p src/generated && pbjs -t static-module -w es6 -o src/generated/protobuf.js ../specs/card_activate_model.proto --force-long && pbts -o src/generated/protobuf.d.ts src/generated/protobuf.js",
+    "generate-protobuf:win32": "mkdir src\\generated | pbjs -t static-module -w es6 -o src\\generated\\protobuf.js ..\\specs\\card_activate_model.proto --force-long | pbts -o src\\generated\\protobuf.d.ts src\\generated\\protobuf.js",
```

Comment:
Should we remove that? Only difference is backslashes. WSL should handle that.

Reply:
I'd just keep it for now until it breaks something for us.